s3-source-connector/src/integration-test/java/io/aiven/kafka/connect/s3/source/IntegrationTest.java

```diff
@@ -384,18 +367,4 @@ public static void saveToS3(final String bucketName, final String folderName, fi
         final PutObjectRequest request = new PutObjectRequest(bucketName, folderName + fileNameInS3, fileToWrite);
         s3Client.putObject(request);
     }
-
-    public void multipartUpload(final String bucketName, final String key) {
-        try (S3OutputStream s3OutputStream = new S3OutputStream(bucketName, key, S3OutputStream.DEFAULT_PART_SIZE,
-                s3Client);
-                InputStream resourceStream = Thread.currentThread()
-                        .getContextClassLoader()
-                        .getResourceAsStream(S3_FILE_NAME)) {
-            assert resourceStream != null;
-            final byte[] fileBytes = IOUtils.toByteArray(resourceStream);
-            s3OutputStream.write(fileBytes);
-        } catch (IOException e) {
-            LOGGER.error(e.getMessage());
-        }
-    }
```

Comment:
Why we are removing this?

Reply:
My comment in the previous PR is that we seemed to be testing upload to S3 functionality in the source connector. I didn't see the need for that particular test, as we should be testing files that are on S3.

Reply:
Ah, got it. For some reason thought that it is a production code. Good catch, we definitely need to test the connector, not the S3 API.